app/package.json

```diff
@@ -8,6 +8,9 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.5.3",
```

Comment:
we already have Tailwind and DaisyUI. I would stick to using those rather than adding more UI libraries. You were already using them for loading states earlier, what issue did you run into that made you want to use these libraries?

Reply:
Mainly the daisy ui loading components are not good looking for Loading. So i decided to go with this.

Reply:
@MikaelCarpenter  NVM I found one example to create that on tailwind, i will copy that code for loading. and get rid of material ui

Reply:
awesome! go ahead and merge after that 👍